server/models/FiltersModel/index.js

```diff
@@ -0,0 +1,43 @@
+import { STUDIES_TO_OMIT, TRUE_STRING, FALSE_STRING } from '../../constants'
+
+const FiltersModel = {
+  calculateFilters: function (filters, userAccess) {
+    if (!filters) {
+      this.defaultFilters.sites = userAccess.filter(
+        (site) => !STUDIES_TO_OMIT.includes(site)
+      )
+
+      return this.defaultFilters
+    } else {
+      filters.sites = Object.values(filters.sites).map(({ value }) => value)
+
+      return filters
+    }
+  },
+  defaultFilters: {
+    chrcrit_part: [
+      { name: 'HC', value: TRUE_STRING },
+      { name: 'CHR', value: TRUE_STRING },
+      { name: 'Missing', value: TRUE_STRING },
+    ],
+    included_excluded: [
+      { name: 'Included', value: TRUE_STRING },
+      { name: 'Excluded', value: FALSE_STRING },
+      { name: 'Missing', value: FALSE_STRING },
+    ],
+    sex_at_birth: [
+      { name: 'Male', value: TRUE_STRING },
+      { name: 'Female', value: TRUE_STRING },
+      { name: 'Missing', value: TRUE_STRING },
+    ],
+    sites: [],
+  },
+  normalizeFilters: (filters) => {
+    return {
+      ...filters,
+      sites: filters.sites.map((site) => ({ label: site, value: site })),
```

Comment:
should we standardize on a format for the sites array, either using an array of strings or an array of label/value objects?

Reply:
Not for this, we have two distinct dependencies and they work differently. Mongo requires an array of strings, and our form on the UI requires an object with a label and a value.

Reply:
are we saving these filters to the database?

Reply:
no, the $in property from mongo only takes a string array